.devcontainer/devcontainer.json

```diff
@@ -1,17 +1,17 @@
 {
-  "image": "mcr.microsoft.com/devcontainers/base",
+  "name": "Node.js",
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:16",
+  "features": {
+    "ghcr.io/devcontainers-contrib/features/pnpm:2": {}
+  },
   "customizations": {
     "codespaces": {
       "openFiles": ["README.md", "CONTRIBUTING.md"]
     },
     "vscode": {
-      "extensions": [
-        "lizebang.bash-extension-pack",
-        "EditorConfig.EditorConfig",
-        "GitHub.vscode-github-actions"
-      ]
+      "extensions": ["EditorConfig.EditorConfig"]
     }
   },
-  // https://webinstall.dev/
-  "postCreateCommand": "curl -sS https://webi.sh/shfmt | sh && curl -sS https://webi.sh/shellcheck | sh"
+  "postCreateCommand": ".devcontainer/postCreate.sh",
```

Comment:
```diff
-  "postCreateCommand": ".devcontainer/postCreate.sh",
+  "postCreateCommand": "pnpm install",
```

Reply:
Just a bit of future-proofing for when more post-create commands need to be added. Plus, it's a cheat to add a bit more language diversity to this project :wink: